src/azul/terra.py

```diff
@@ -91,129 +93,119 @@
 log = logging.getLogger(__name__)
 
 
+class SourceType(enum.Enum):
```

Comment:
```suggestion
class TDRSourceType(enum.Enum):
```

or just `Type` and nest in TDRSourceSpec